src/blob_storage.cc

```diff
@@ -45,6 +45,11 @@ void BlobStorage::ExportBlobFiles(
 void BlobStorage::AddBlobFile(std::shared_ptr<BlobFileMeta>& file) {
   WriteLock wl(&mutex_);
   files_.emplace(std::make_pair(file->file_number(), file));
+  AddStats(stats_, cf_id_, TitanInternalStats::LIVE_BLOB_SIZE,
+           file->file_size());
```

Comment:
seems it doesn't follow the definition `total blob value size referenced by LSM tree`. `file_size` includes header + footer + key